testing-common/src/main/java/io/opentelemetry/instrumentation/testing/junit/http/HttpServerTestOptions.java

```diff
@@ -29,15 +31,22 @@ public final class HttpServerTestOptions {
                   SemconvStabilityUtil.getAttributeKey(SemanticAttributes.NET_PEER_PORT))));
 
   public static final SpanNameMapper DEFAULT_EXPECTED_SERVER_SPAN_NAME_MAPPER =
-      (uri, method, route) -> route == null ? method : method + " " + route;
+      (uri, method, route) -> {
+        if (HttpConstants._OTHER.equals(method)) {
+          method = "TEST";
```

Comment:
if https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/9441 gets merged before you'll have to change this to `HTTP`

Reply:
heads up, I just merged #9441